Update template pages (#25)
Imageomics/Imageomics-guide: docs/wiki-guide/HF_DatasetCard_Template_mkdocs.md

# Dataset Card Template

> Below is the Dataset Card Template for Imageomics and ABC. You can download or copy this content and paste it into a new Markdown file to create a new dataset card. 

Reviewer: ```suggestion
Below are the Dataset Card templates for Imageomics and ABC. You can download or copy the appropriate dataset card content and paste it into a new Markdown file to create a README for your dataset. 
```
